### ISSUE TITLE
Error trying to run test

### ISSUE THREAD

**Issue by scraggo** [reid47/jest-browser#1]
After following setup instructions and running `npm run test`:

```
Validation Error:

  Module jest-browser should have "jest-preset.js" or "jest-preset.json" file at the root.

  Configuration Documentation:
  https://jestjs.io/docs/configuration.html
```

Tried:
Following on-screen instructions, adding `jest-preset.js`
Following jest's docs, adding `jest.config.js`

**Comment by reidmit** [comment]
Thanks for giving this a shot and filing this issue! Unfortunately, I haven't had time to work on this in a while. If you're looking for a project that does the same thing and is more actively maintained, I'd recommend taking a look at this:

https://github.com/smooth-code/jest-puppeteer

That's the library recommended by the Jest docs here:

https://jestjs.io/docs/en/puppeteer

**Comment by piranna** [comment]
Not sure if I'm missing the point, but seems like `jest-puppeteer` allow to write jest tests that control puppeteer, while `jest-browser` just exec the tests in a browser environment instead of the default `jsdom`, isn't it? If so, targets are different and complementary, since `jest-browser` could be used for unit test or integration tests (using a real browser instead of mocks), while `jest-puppeteer` seems to be more focused for end-to-end tests... Am i right?

**Comment by clabe45** [comment]
I agree with @piranna. I think there is a real need for this, but I understand if you don't have the time to work on it. Maybe people looking for repos to contribute to could work on this.

**Comment by reidmit** [comment]
Hi all, thanks for the questions! As I understand it, `jest-puppeteer` does a couple of convenient things:
- Provides a test environment that initializes/cleans up a Puppeteer instance instead of a JSDOM instance (similar to [this example](https://jestjs.io/docs/puppeteer#custom-example-without-jest-puppeteer-preset))
- Provides some helpers for writing expectations in the new Puppeteer environment: https://github.com/smooth-code/jest-puppeteer/blob/master/packages/expect-puppeteer/README.md#api

My experimental repo here also provides a Puppeteer-based test environment and a set of helpers for working with it. The main differences are that the helpers look slightly different, and `jest-puppeteer` has had lots more time/energy/testing put into it 😄 

That said, I'd welcome PRs here to improve this project, but you're also welcome to fork it and improve it that way!
